lib/labels_pdf_renderer.rb

```diff
@@ -0,0 +1,20 @@
+require_relative "./multipage_pdf_renderer"
+
+module LabelsPdfRenderer
```

Comment:
Invoking wkhtmltopdf for each page is awfully slow. It takes several dozen seconds to render a PDF with many pages (up to 109 for a Challenge box).

I'm pretty sure we could render a single HTML and invoke wkhtmltopage once, and still get the same result.

Reply:
See #1658